libs/api/domains/user-profile/src/lib/userProfile.resolver.ts

```diff
@@ -41,6 +43,11 @@ export class UserProfileResolver {
     return this.userUserProfileService.getUserProfileLocale(user)
   }
 
+  @Query(() => ActorProfileResponse)
```

Comment:
So queries needs to be 
```suggestion
  @Query(() => ActorProfileResponse, {name: 'userProfileActorProfiles'})
```

or you can rename the function
@Query(() => ActorProfileResponse)
userProfileActorProfiles(@CurrentUser()......
```

I usually prefer using the option property with `name` as the gql naming can be rather long and I prefer nice typescript function names.